2.6.md

```diff
@@ -2963,6 +2985,40 @@ Information on how often and what triggers an ad slot being refreshed.
   </tr>
 </table>
 
+### 3.2.35 - Object: DurFloors <a name="objectdurfloors"></a>
```

Comment:
Note the new object is given many different names in this document (DurFloors, Durfloors, Durfloor). When we choose one, I recommend the plural form because the object itself contains multiple floors.